### PR TITLE
WT-9025 Add tiered storage scenario to all schema Python tests

### PR DESCRIPTION
**Pull request by mm-ng**

### CODE REVIEW

**Comment by mm-ng** [comment]
@sueloverso It seems like backup and recovery are not supported with tiered storage - there's a failure in test_schema08 when we allow these operations to run. Is this planned to be addressed in a future project or should we create a FIXME ticket for this?

**Review by AlexeyAnisimovAU — APPROVED** [review]
Good job, LGTM!

File: test/suite/test_schema05.py
Comment (sueloverso) [inline test/suite/test_schema05.py:60]: Indexes are like column groups and are not expected to work with tiered storage right now.

**Review by sueloverso — COMMENTED** [review]
Generally seems okay. Tiered does not work with indexes either (nor backup).

**Comment by sueloverso** [comment]
@mm-ng and @AlexeyAnisimovAU  I couldn't really tell how much of each test is "Tiered storage is not supported with...". If pretty much nothing in these tests runs with tiered it may not be worth doing this set of changes. I'll let you decide.

**Comment by AlexeyAnisimovAU** [comment]
> @mm-ng and @AlexeyAnisimovAU I couldn't really tell how much of each test is "Tiered storage is not supported with...". If pretty much nothing in these tests runs with tiered it may not be worth doing this set of changes. I'll let you decide.

@sueloverso This is a valid question. Thank you! We checked that after we enabled tiered storage scenarios we skip a lot of them, but we also run some. Also, I believe that by merging this change we will make life easier for future developers who will be adding support of column groups, indexes, and LSM. They will just modify the skipping logic in each test.
Please let us know what you think.

**Review by sueloverso — APPROVED** [review]
LGTM - thanks for checking my question.

**Comment by sueloverso** [comment]
Please check the failure. If it is 1. a known issue add something to a ticket, 2. a new unrelated issue, please create a ticket, or 3. related to these changes, please fix it before merging.

**Comment by AlexeyAnisimovAU** [comment]
Thank you Sue for checking the evergreen status! This is a known issue, not related to this change: [WT-8987](https://jira.mongodb.org/browse/WT-8987)

**Pull request merged**
